src/bootstrap/lib.rs

```diff
@@ -129,7 +129,8 @@ const EXTRA_CHECK_CFGS: &[(Option<Mode>, &'static str, Option<&[&'static str]>)]
     /* Extra values not defined in the built-in targets yet, but used in std */
     (Some(Mode::Std), "target_env", Some(&["libnx"])),
     // (Some(Mode::Std), "target_os", Some(&[])),
-    (Some(Mode::Std), "target_arch", Some(&["asmjs", "spirv", "nvptx", "xtensa"])),
+    // #[cfg(bootstrap)] loongarch64
```

Comment:
No, cf. this comment earlier https://github.com/rust-lang/rust/pull/96971#issuecomment-1192610990 (which you even replied :smile:)

*In summary the `cfg(bootstrap)` is just here help the people who does the stage0 bump, so that they know that they should remove what is following, here loongarch64*